File: PythonProject/Week 2/models/student.py
class Student:
    def __init__(self, student_id, name, email, dept_id, stream_id, enrollment_year, current_semester):
        self.student_id = student_id
        self.name = name
        self.email = email
        self.dept_id = int(dept_id)
        self.stream_id = stream_id
        self.enrollment_year = int(enrollment_year)
        self.current_semester = int(current_semester)

    def __repr__(self):
        return f"Student({self.student_id}, {self.name}, Dept:{self.dept_id})"

File: Java/Week_3-4/src/models/Student.java
package models;

public class Student
{
    private final String studentID;
    private final String name;
    private final String streamID;
    private final String deptID;
    private final int enrollmentYear;

    public Student(String studentID, String name, String streamID, String deptID, int enrollmentYear) {
        this.studentID = studentID;
        this.name = name;
        this.streamID = streamID;
        this.deptID = deptID;
        this.enrollmentYear = enrollmentYear;
    }

    public String getStudentID() {return studentID;}
    public String getName() {return name;}
    public String getStreamID() {return streamID;}
    public String getDeptID() {return deptID;}
    public int getEnrollmentYear() {return enrollmentYear;}

}
